packages/tradershub/src/features/cfd/screens/EnterPassword/EnterPassword.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { useActiveTradingAccount } from '@deriv/api';
+import { Button, Text, TextField, useBreakpoint } from '@deriv/quill-design';
+import { TMarketTypes, TPlatforms } from '../../../../types';
+import { validPassword } from '../../../../utils/password';
+import { MarketTypeDetails, PlatformDetails } from '../../constants';
+
+type TProps = {
+    isLoading?: boolean;
+    marketType: TMarketTypes.CreateOtherCFDAccount;
+    onPasswordChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    onPrimaryClick?: () => void;
+    onSecondaryClick?: () => void;
+    password: string;
+    passwordError?: boolean;
+    platform: TPlatforms.All;
+};
+
+/**
+ * Component to display the enter password screen
+ * @param {boolean} isLoading - loading state
+ * @param {TMarketTypes.CreateOtherCFDAccount} marketType - market type all or synthetic or financial
+ * @param {Function} onPasswordChange - callback to handle password change
+ * @param {Function} onPrimaryClick - callback to handle primary button click
+ * @param {Function} onSecondaryClick - callback to handle secondary button click
+ * @param {string} password - password value
+ * @param {boolean} passwordError - password error state
+ * @param {TPlatforms.All} platform - platform Mt5 or Dxtrade
+ * @returns {React.ReactNode} - returns the enter password screen component
+ */
+
+const EnterPassword: React.FC<TProps> = ({
+    isLoading,
+    marketType,
+    onPasswordChange,
+    onPrimaryClick,
+    onSecondaryClick,
+    password,
+    passwordError,
+    platform,
+}) => {
+    const { isDesktop } = useBreakpoint();
+    const title = PlatformDetails[platform].title;
+    const { data } = useActiveTradingAccount();
+    const accountType = data?.is_virtual ? 'Demo' : 'Real';
+    const marketTypeTitle =
+        platform === PlatformDetails.dxtrade.platform ? accountType : MarketTypeDetails[marketType].title;
+
+    return (
+        <div className='flex ps-800 w-full lg:inline-flex lg:w-[400px] lg:pt-1000 lg:pb-1200 lg:px-[24px] flex-col justify-center items-start rounded-400 border-sm bg-system-light-primary-background'>
```

Comment:
Using tailwind classes like this makes tracking the styling very hard. IMHO we need to create custom classes to make our styling more readable.

Reply:
Implemented as mentioned by @aizad-deriv 
There are no scss or css file in the tradershub.
need input on this approach whether to use inline css or css?

Reply:
That's true @hirad-deriv it is quite confusing however I do believe this the way to go. Once, Tradershub design is fully translated to semantic values it will be lesser and more cleaner.

In terms of tracking components in React there's numerous ways, one of them is `React devtools`, chrome extension. 

It's far from perfect but I feel we are currently on the right path to make things better 😃 🙏🏼